school_library.rb

```diff
@@ -0,0 +1,29 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- Good job done so far, but the Person class should have an `id` instance variable which could be a random number when the person class is created. Please fix this 🙏

Reply:
@Noted the changes will be implemented on the future commits. :slightly_smiling_face:

Reply:
- [ ] Hi Please as mentioned by the previous reviewer and as part of the requirements, kindly add the `id` instance variable to your `Person` class